pkg/cmd/inspect/inspect.go

```diff
@@ -105,6 +115,10 @@ func NewCmdInspect(streams genericclioptions.IOStreams) *cobra.Command {
 func (o *InspectOptions) Complete(cmd *cobra.Command, args []string) error {
 	o.args = args
 
+	if o.defaults == true {
+		o.args = append(o.args, defaults...)
```

Comment:
I guess we should not worry much about `--defaults` adding duplicate resources since we keep track of collected data via the *resourceContext. However this does mean that if a user does something like `must-gather clusteroperators --defaults`, we'd be fetching clusteroperators twice